docs/docs/guides/develop/integrate.md

```diff
@@ -0,0 +1,129 @@
+---
+title: Integrate
+titleTemplate: ":title | Develop | Tuist"
+description: Learn how to use Tuist in your CI workflows.
+---
+
+# Integrate
```

Comment:
I would call this `Continous integration` instead. `Integrate` is too generic and in the guide, we only reference Continuous integration.

Reply:
I renamed it to `Continuous Integration`, and placed it under a section called `Automate`, which leaves space for the future work that's coming:

![Uploading image.png…]()